converters/fromZigbee.js

```diff
@@ -5561,14 +5561,33 @@ const converters = {
             return payload;
         },
     },
+    tuya_color_colortemp: {
+        cluster: 'lightingColorCtrl',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+
+            if (msg.data.hasOwnProperty('colorTemperature')) {
+                const value = Number(msg.data['colorTemperature']);
+                // Mapping from
+                // Warmwhite 0 -> 255 Coldwhite
+                // to Homeassistant: Coldwhite 153 -> 500 Warmwight
+                //
+                //
+                result.color_temp = Math.round(-1.36 * value + 500);
+            }
+
+            return result;
+        },
+    },
     tuya_led_controller: {
         cluster: 'lightingColorCtrl',
         type: ['attributeReport'],
```

Comment:
Please change this to `['attributeReport', 'readResponse'],` and merge the `tuya_color_colortemp` with this.